### ISSUE TITLE
REQUEST: Request a Learning session for Tetragon

### ISSUE THREAD

**Issue by mtardy** [kubernetes/sig-security#90]
**Please tell us a bit more about the topic** 

I would love to present [Tetragon](https://github.com/cilium/tetragon) to people at SIG security tooling! It's an eBPF-based Security Observability and Runtime Enforcement software. In brief, it allows for process lifecycle observability by default and you can extend its capabilities by writing Kubernetes CRD in YAML (called TracingPolicy) that will be translated to eBPF programs to perform customized observability and enforcement.

I'm not exactly sure from which angle we want to present but I have some ideas :)!

**Please share speaker details**

Partially unsure.
I think I will present but would love to have someone from the Tetragon team with me.

**If speaker is confirmed, please share speaker availability** 

The next challenge will be to find a nice timeslot for the learning session!

**Explain the benefits to the community that this session provides**

The community would benefit from learning about this new open-source tool that solves security observability and enforcement problem. People are starting to realize that a lot of their security needs can be addressed with eBPF-powered software, but may not know how to get started. Introducing attendees to Tetragon and its TracingPolicy gives them an opportunity to get a bit familiar with eBPF security coming straight from the kernel through things they already know like Kubernetes custom resources. In addition, it will introduce them to some kernel concepts that they will need to understand for security, like syscalls, tracepoints, kprobes, etc, and how to leverage them in the context of Kubernetes clusters.

See you SIG security people!! 👋

**Comment by PushkarJ** [comment]
@mtardy thanks for filing this! Would May 24 work for you and the tetragon team?

**Comment by PushkarJ** [comment]
We are postponing this due to my unavailability to host on May 24. I am working with @mtardy to figure out the next date and will post an update here when I know more.

**Comment by PushkarJ** [comment]
@mtardy can you and the team present it for June 21 slot?

**Comment by mtardy** [comment]
> @mtardy can you and the team present it for June 21 slot?

Personally yes, I'll propose and double confirm.

**Comment by PushkarJ** [comment]
@mtardy any luck confirming this yet with the rest of the team?

**Comment by mtardy** [comment]
> @mtardy any luck confirming this yet with the rest of the team?

Yep, sorry for the late reply, I think it's not for the best because one of my team mate cannot join. Do you think we can do bpfd at this date and then tetragon maybe?

**Comment by mtardy** [comment]
I think the 19th of July would be good.

**Comment by PushkarJ** [comment]
Let's do it @mtardy ! Tetragon is on for July 19th. 

Thanks for your patience with this :)

**Comment by iamvolvo** [comment]
Kind reminder to update the issue with a link to the recording when available :-)

**Comment by PushkarJ** [comment]
@tabbysable can you please add a link to July 19 2023 Tooling session as a comment?

/remove-lifecycle stale

**Comment by mtardy** [comment]
/reopen

Might be a bit outdated now but I think we still don't have the link.

**Comment by tabbysable** [comment]
I think you probably have the link now! thanks for keeping track of this and your patience with the upload backlog!

https://www.youtube.com/watch?v=4ifEI1n4lY4
